app/assets/images/TravelGo_files/polygon.self-ceb351c472231550371b47497b02ad128fbf6e44c30482fb5c153ef880fad696.js

```diff
@@ -0,0 +1,29 @@
+(function() {
+  var extend = function(child, parent) { for (var key in parent) { if (hasProp.call(parent, key)) child[key] = parent[key]; } function ctor() { this.constructor = child; } ctor.prototype = parent.prototype; child.prototype = new ctor(); child.__super__ = parent.prototype; return child; },
```

Comment:
Line is too long.<br>Expected '{' and instead saw 'child'.<br>A constructor name should start with an uppercase letter.